functions/src/express_configs/express_portal.ts

```diff
@@ -109,6 +110,37 @@ function extractAuth0Fields(request: Request, response: Response, next: () => vo
 }
 app.use(extractAuth0Fields);
 
+app.use(
+  "/auth0/vanity/create",
```

Comment:
```suggestion
  "/gsuite/vanity/create",
```

For the backend, if the route starts with `/auth0` anyone who has signed in to the portal will be able to use this API endpoint. If it starts with `/gsuite` then only officers can make use of this endpoint.